purchases/src/main/kotlin/com/revenuecat/purchases/models/StoreTransaction.kt

```diff
@@ -70,9 +71,9 @@ data class StoreTransaction(
     val originalJson: JSONObject,
 
     /**
-     * Offering that was presented when making the purchase. Always null for restored purchases.
+     * Context of the offering that was presented when making the purchase.
      */
-    val presentedOfferingIdentifier: String?,
+    val presentedOfferingContext: PresentedOfferingContext?,
```

Comment:
I had doubts on this... Note that it's nullable here but not nullable in the `Package`/`StoreProduct`/`SubscriptionOption`. It will be null here when it can't find any information about the context (usually restores and such) vs having a context with empty data when the origin of the purchase is through a product/we don't have offering data.

Lmk if you think we should standardize this somehow and we can chat, but I thought it was ok like this for now.

Reply:
I think this is fine to be nullable here! Forcing an object before the transaction makes it a cleaner (especially when copying the object down into subscription options. I probably would have done the same here in making it nullable in the transaction 👍